server/server_test.go

```diff
@@ -386,3 +386,44 @@ func TestNilMonitoringPort(t *testing.T) {
 		t.Fatal("HttpAddr should be nil.")
 	}
 }
+
+type DummyAuth struct{}
+
+func (d *DummyAuth) Check(c ClientAuthentication) bool {
+	return c.GetOpts().Username != "test"
```

Comment:
When I was suggesting using `c.GetOpts().Username` was if you wanted to be more sophisticated and have a test that tries to connect with an allowed user name and one without.

Reply:
done